src/engine/sync/internalclock.cpp

```diff
@@ -12,16 +12,17 @@
 
 namespace {
 const mixxx::Logger kLogger("InternalClock");
+constexpr mixxx::Bpm kDefaultBpm(124.0);
```

Comment:
we've talked about having a mixxx-wide default bpm, have we established one yet?  (not needed for this PR)

Reply:
I'm not aware we have. We can probably add it later on.